style.css

```diff
@@ -721,3 +721,143 @@ button:active {
     display: none;
   }
 }
+
+/* -------CSS FOR POPUP MOBILE    */
+
+.wrap-pop {
+  background-color: rgb(0, 0, 0, 0.5);
+  display: none;
+  transition: 500ms ease-in-out;
+  padding: 2%;
+  height: 95vh;
+  border: 1px solid #ddd;
+  overflow-y: scroll;
+}
+
+.wrap-pop.active {
+  display: flex;
+  justify-content: center;
+  position: fixed;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  z-index: 10;
+}
+
+.popup-mb {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  padding: 4%;
+  margin: 2%;
+  transition: 200ms ease-in-out;
+  overflow-y: scroll;
+  overflow: hidden;
+}
+
```

Comment:
- [ ] Kindly for the mobile version, I can't see the whole paragraph and the two buttons. Kindly make them visible or add scroll.

Your design
![image](https://user-images.githubusercontent.com/35535130/175435608-aae05e62-5e28-4758-997b-a5e1e870a1ea.png)

Original design
![image](https://user-images.githubusercontent.com/35535130/175435663-31b31b64-4252-4918-bda7-89ba0f4a1599.png)